### DISCUSSION TITLE
Can you vary something more than 'Vary Subtle' but less than 'Vary strong'?

### DISCUSSION THREAD

**Discussion by Kh4rj1** [lllyasviel/Fooocus#1998] [General]
Is there like a hidden advanced setting for how much these actually vary the image, or is it more like two different models that you can't really put on a continuous slider? 
Because often I find myself wanting to vary something, well, more than subtely but not quite so strongly :)

Thanks!

**Comment by poor7** [comment]
I think it's here: Advanced - Developer Debug Mode - Debug Tools - Forced Overwrite of Denoising Strength of "Vary" 

<img width="411" alt="Безымянный" src="https://github.com/lllyasviel/Fooocus/assets/154618384/d6893dcf-369e-4503-9a78-033397f903fd">

**Reply by Kh4rj1** [reply]
Yes that seems to be it, thanks! Do you happen to know what values the normal subtle and strong correspond to?

edit: nvm I figured it out with trial and error, Vary Subtle means 0,5 and Vary Strong means 0,85
